scripts/tools/release-cli/release-cli.js

```diff
@@ -38,6 +38,7 @@ const pkgJsonPaths = {
   core: path.resolve(base, 'core', 'public', 'package.json'),
```

Comment:
I think there might be some issues with regards to the nextVersion generated for nightly Container.

So far nightlies have been done for the `core/client/etc` packages where we follow the `core/package.json` version as a basis + "-dev" + timestamp. 

Specficially [here](https://github.com/SAP/luigi/blob/d039ff2b364cc88ef25001763745bd5fc2cca00f/scripts/tools/release-cli/release-cli.js#L89-L91)
```
function getNextVersion() {
  return semver.inc(getVersion('core'), 'patch');
}
```

In this PR I think this would end up (due to [this line](https://github.com/SAP/luigi/blob/main/scripts/tools/release-cli/release-cli.js#L149)) publishing Container nightlies with 2.15.0-dev.2024.... It should instead look sth like `"1.3.0-dev.2024"`

Reply:
One solution might be 

https://github.com/SAP/luigi/blob/d039ff2b364cc88ef25001763745bd5fc2cca00f/scripts/tools/release-cli/release-cli.js#L180-L182

```
 for (const name of Object.keys(pkgJsonPaths)) {
   // check name or path

   if (name is related to container/package.json && Nightly = true) {
      const containerNightly = modify the version accordingly;
        writeVersion(pkgJsonPaths[name], <containerNightly here>);
    } else {
        writeVersion(pkgJsonPaths[name], input.version);
    }
  }
```